quickannotator/db/crud/annotation.py

```diff
@@ -122,22 +168,63 @@ def insert_annotations(self, polygons: List[BaseGeometry], tile_ids: List[int] |
 
     # READ
     def get_annotation_by_id(self, annotation_id: int) -> db_models.Annotation:
-        result = get_annotation_query(self.model, 1/self.scaling_factor).filter_by(id=annotation_id).first()
+        result = self.query.filter_by(id=annotation_id).first()
         return result
 
+
     def get_annotations_by_ids(self, annotation_ids: List[int]) -> List[db_models.Annotation]:
-        result = get_annotation_query(self.model, 1/self.scaling_factor).filter(self.model.id.in_(annotation_ids)).all()
+        result = self.query.filter(self.model.id.in_(annotation_ids)).all()
         return result
 
+
     def get_annotations_for_tiles(self, tile_ids: List[int]) -> List[db_models.Annotation]:
-        result = get_annotation_query(self.model, 1/self.scaling_factor).filter(self.model.tile_id.in_(tile_ids)).all()
+        result = self.query.filter(self.model.tile_id.in_(tile_ids)).all()
+        return result
+    
+
+    def get_all_annotations(self) -> List[models.Annotation]:
+        result = self.query.all()
         return result
 
+
     def get_annotations_within_poly(self, polygon: BaseGeometry) -> List[db_models.Annotation]:
         scaled_polygon = self.scale_polygon(polygon, self.scaling_factor)
         # NOTE: Sqlite may not use the spatial index here.
-        result = get_annotation_query(self.model, 1/self.scaling_factor).filter(func.ST_Intersects(self.model.polygon, func.ST_GeomFromText(scaled_polygon.wkt, 0))).all()
+        result = self.query.filter(func.ST_Intersects(self.model.polygon, func.ST_GeomFromText(scaled_polygon.wkt, 0))).all()
         return result
+    
+
+    def export_all_annotations_to_tar(self, tarpath: str):
+        """
+        Saves all annotations to a tar archive.
+
+        Args:
+            tarname (str): The name of the tar file to save the annotations to.
+        """
+        with tarfile.open(tarpath, mode='w') as tar:
```

Comment:
wrap in a gzip

Reply:
you can see some examples here: https://andrewjanowczyk.com/serialization-and-storage-of-geojson-for-in-digital-pathology/

Reply:
actually, you may be able to just use:

>>> fp = tarfile.open("zen.tar.gz","w:gz")

Reply:
done.